swagger.yml

```diff
@@ -2351,6 +2369,39 @@ definitions:
         type: object
       branch:
         type: string
+  buildStatus:
+    type: object
+    properties:
+      active:
+        type: integer
+      pending_concurrency:
+        type: integer
+      enqueued:
+        type: integer
+      build_count:
+        type: integer
+      minutes:
+        type: object
+        properties:
+          current:
+            type: integer
+          current_average_sec:
+            type: integer
+          previous:
+            type: integer
+          period_start_date:
+            type: string
+            format: dateTime
+          period_end_date:
+            type: string
+            format: dateTime
+          last_updated_at:
+            type: string
+            format: dateTime
+          included_minutes:
+            type: string
+          included_minutes_with_packs:
```

Comment:
I have some minor mixed feeling about adding these "included" two fields here (you can see more in https://github.com/netlify/bitballoon/pull/5881 - tl;dr I didn't wanna add them in this endpoint), but it's less likely to change so... I guess it's okay to be as-is